vumi/middleware/tests/test_message_storing.py

```diff
@@ -15,7 +15,8 @@ def setUp(self):
 
         # Create and stash a riak manager to clean up afterwards, because we
         # don't get access to the one inside the middleware.
-        self.persistence_helper.get_riak_manager()
+        manager = self.persistence_helper.get_riak_manager()
```

Comment:
There are places where we patch in `persistence_helper.get_riak_manager()` to replace the code under test's own manager creation, and we need to make sure that those also do their own cleanup.